notebooks/book2/29/kf_parallel.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            ax.plot(*xs[n,:,:2].T, ls="--", color=f"C{n}",linewidth=0.7 if is_latexify_enabled() else 1.5)
</pre><p>In case of multiple linewidth</p><p>you can use <code>LINE_WIDTH1</code>, <code>LINE_WIDTH2</code> , etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>